package/boot/uboot-imx/Makefile

```diff
@@ -42,10 +42,19 @@ define U-Boot/wandboard
   BUILD_DEVICES:=wandboard_dual
 endef
 
+define U-Boot/pico-pi-imx7d
```

Comment:
@Leo-PL please, manage alphabetical order of devices definitions.

Reply:
Fixed.